docs/connectors/taps/adwords.rst

```diff
@@ -0,0 +1,81 @@
+
+.. _tap-adwords:
```

Comment:
Little history: tap adwords was never really tested and used as a prototype tap to generate and run PPW commands in the beginning of the time; and that's why it was always missing from the doc.

 It does some job however so probably worth documenting it.

Reply:
> It does some job however so probably worth documenting it.

Yeah, I want to add it because it's amongst the singer connectors in PPW but not in the doc and that has confused some users in external Slack channel.
Plus, it has a user base so might as well document it.